Created jonell.json

lowqualityloey opened the pull request:
<!--
YOU MUST FILL OUT THIS TEMPLATE ENTIRELY FOR YOUR PR TO BE ACCEPTED, NONE OF IT IS OPTIONAL UNLESS SPECIFIED.
-->

# Requirements
<!-- Your domain MUST pass ALL the requirements below, otherwise it WILL BE DENIED. -->

<!-- Change each checkbox to [x] (all lowercase, with no spaces between the brackets) to mark it as completed. -->

- [x] I have **read** and **agreed** to the [Terms of Service](https://is-a.dev/terms). <!-- Your request MUST follow the TOS to be approved. -->
- [x] My file is following the [domain structure](https://docs.is-a.dev/domain-structure/).
- [x] My website is **reachable** and **completed**. <!-- We do not permit simple "Hello, world!" or simply copied/mostly blank templated websites. -->
- [x] My website is **software development** related. <!-- Only your root subdomain needs to meet this requirement. -->
- [x] My website is not for commercial use. <!-- Your website's purpose should not be to generate any form of revenue. -->
- [x] I have provided sufficient contact information in the `owner` key. <!-- Provide your email in the `email` field or another platform (e.g. X/Twitter or Discord) for contact. -->
- [x] I have provided a preview of my website below. <!-- This step is required for your domain to be approved. -->

# Website Preview
<!-- Provide a link or screenshot of your website below. -->
https://lowqualityloey.github.io/

lowqualityloey committed:
Add configuration for lowqualityloey GitHub Pages

lowqualityloey committed:
Add subdomain JSON for lowqualityloey GitHub Pages

Created _github-pages-challenge-lowqualityloey.subdomain.json to configure subdomain ownership and TXT record for lowqualityloey's GitHub Pages challenge.

lowqualityloey committed:
Fix email formatting in subdomain config

Removed unnecessary brackets from the owner's email address in the _github-pages-challenge-lowqualityloey.subdomain.json file to ensure proper formatting.

STICKnoLOGIC reviewed (changes requested):
read and followw-> https://docs.is-a.dev/guides/github-pages/

lowqualityloey committed:
Remove old domain and rename subdomain file

Deleted the _github-pages-challenge-lowqualityloey domain file and renamed the subdomain JSON file to _github-pages-challenge-lowqualityloey.jonell.json for improved clarity and organization.

lowqualityloey committed:
Merge branch 'main' into main

lowqualityloey commented on domains/_github-pages-challenge-lowqualityloey:
hi my bad i just made an update now. check it if you like

STICKnoLOGIC replied:
@lowqualityloey , you are missing the root subdomian "your-chosen-subdomain.json" kindly read and follow the docs

iostpa reviewed (changes requested):
You also need to make a file for "jonell.is-a.dev".
